tests/voq/voq_helpers.py

```diff
@@ -410,12 +411,20 @@ def get_eos_mac(nbr, nbr_intf):
     Returns:
         A dictionary with the mac address and shell interface name.
     """
-    if "port-channel" in nbr_intf.lower():
-        # convert Port-Channel1 to po1
-        shell_intf = "po" + nbr_intf[-1]
+
+    if isinstance(nbr['host'], EosHost):
+        if "port-channel" in nbr_intf.lower():
+            # convert Port-Channel1 to po1
+            shell_intf = "po" + nbr_intf[-1]
+        else:
+            # convert Ethernet1 to eth1
+            shell_intf = "eth" + nbr_intf[-1]
     else:
-        # convert Ethernet1 to eth1
-        shell_intf = "eth" + nbr_intf[-1]
+        if "port-channel" in nbr_intf.lower():
+            # convert Port-Channel1 to po1
```

Comment:
In the comment, should change po1 to PortChannel1

Reply:
Fixed